libnavigation-core/build.gradle

```diff
@@ -30,6 +30,13 @@ android {
         buildConfigField 'String', "NAV_NATIVE_SDK_VERSION", String.format("\"%s\"", project.ext.version.mapboxNavigator)
     }
 
+    lintOptions {
+        // the NullSafeMutableLiveData rule fails with index out of bounds exception analysing the MapboxNavigationSDK.kt.
+        // Try removing the line after updating androidx.lifecycle libraries
+        disable "NullSafeMutableLiveData"
```

Comment:
what do you think if we move rules to `lint.xml` file and keep `gradle` file cleaner

Reply:
for some reason, I don't know why, `NullSafeMutableLiveData` stopped crashing Lint 😮

Reply:
Can we disable a rule in `lint.xml` file so that it doesn't run? I found only examples of ignoring issues https://developer.android.com/studio/write/lint#sample-lint.xml-file

Reply:
What is the difference between disabling the rule and ignoring it?
I tried locally with ignoring `NullSafeMutableLiveData` in lint.xml file and it worked.

Reply:
@Zayankovsky , you're right, it disabling and ignoring means the same in this context. Fixed, thanks 👍